example_dropin_environments/julia_mlj/Dockerfile

```diff
@@ -24,4 +24,6 @@ ENV WITH_ERROR_SERVER=1
 #Uncomment the following line to switch to production mode
 #ENV PRODUCTION=1 MAX_WORKERS=4 SHOW_STACKTRACE=1
 
+USER nobody
```

Comment:
This might not matter too much. In LRS we run the container with a specific set of user id / group id (defaults are 1000:1000), also we force chown on the /opt/code to these.
I'm not sure if setting USER to something else would do it.

Reply:
can you test building such image and testing on staging and/or locally?

Reply:
not sure I can test this on staging before merging the PR. Will try

Reply:
@klichukb I tested this image on Staging. It really lacks permissions if running for any non-root user (when tried to install model dependencies in the exec env).
The only solution to this is to grant permissions to all users for the `$HOME` directory which is "/opt". That directory keeps pip cache, virtualenvs the the code.

We can't know which user will run this container but we definitely can't run the containers from `root`. Chainguard will never let us to use `1000` as a user - there is no way to create custom users, it only gives us `nonroot` user that we can use when running the container - that's requirement from STIG/FIPS compliance.

Reply:
Right, for that reason I updated all the Dockerfiles in this PR.